django_socio_grpc/proto_serializers.py

```diff
@@ -76,59 +85,59 @@ def populate_dict_with_none_if_not_required(self, data_dict, message=None):
             )
 
         is_update_process = (
-            hasattr(self.Meta, "model") and self.Meta.model._meta.pk.name in data_dict
+            hasattr(self.Meta, "model") and get_model_pk(self.Meta.model).name in data_dict
         )
+        clean_dict = {}
```

Comment:
can you add a comment to explain why we need to instanciate a new object ?

Reply:
to keep the method functional